packages/scss/src/components/status/index.scss

```diff
@@ -0,0 +1,6 @@
+@use 'exports' as *;
+
+.status {
+	@include vars;
```

Comment:
it would be interested to add errors, warning and pending states

Reply:
Done.